pages/account/manage/event/[[...data]].js

```diff
@@ -332,6 +344,16 @@ export default function ManageEvent({ BASE_URL, event }) {
                       maxLength="16"
                     />
                   </div>
+                  <div className="col-span-3 sm:col-span-4">
+                      <TagsInput
+                        onTagAdd={handleTagAdd}
+                        onTagRemove={handleTagRemove}
+                        tags={tags}
+                      />
+                      <p className="text-sm text-primary-medium-low dark:text-primary-low-high">
+                        Separate tags with commas.
+                      </p>
+                    </div>
```

Comment:
@amandamartin-dev it's highly unlikely from me, based on the changes I did to the JSX, this is all the change for the add event component that I made and this change is highly needed and I just copied it from the add profile component so I can maintain consistency. Could also be the problem but I don't see it

Reply:
Great collaboration everyone 💪 

@Dun-sin if you comment out these lines does the problems still exist or does it go away? This will help us isolate the problem to fix, I just updated your PR with the latest changes

Reply:
After some debugging (`main` was working fine), the css classes have been brought over directly from the manage profile page, which breaks the layout on the manage event page, this can be fixed by using the styles the other form fields use currently on the manage event page...

Before suggested changes...

<img width="1674" alt="Screenshot 2023-10-14 at 08 31 17" src="https://github.com/EddieHubCommunity/BioDrop/assets/624760/5de6fbbc-ceb5-4227-ae65-e6a49c44065b">

```suggestion
                  <div className="mt-1 sm:col-span-2 sm:mt-0">
                    <TagsInput
                      onTagAdd={handleTagAdd}
                      onTagRemove={handleTagRemove}
                      tags={tags}
                    />
                    <p className="text-sm text-primary-medium-low dark:text-primary-low-high">
                      Separate tags with commas.
                    </p>
                  </div>
```

After suggested changes...

<img width="1665" alt="Screenshot 2023-10-14 at 08 31 39" src="https://github.com/EddieHubCommunity/BioDrop/assets/624760/339ca3f4-617c-465e-b20e-10028697f331">